src/sagemaker/pytorch/estimator.py

```diff
@@ -199,6 +214,9 @@ def __init__(
             if self.framework_version and Version(self.framework_version) >= Version("1.3"):
                 kwargs["enable_sagemaker_metrics"] = True
 
+        if "instance_type" in kwargs:
+            self.instance_type = kwargs["instance_type"]
```

Comment:
is `self.instance` already be assigned in the base class's init func?

Reply:
Yes, removed.